packages/components/Separator/src/Separator.types.ts

```diff
@@ -1,11 +1,26 @@
 import { ViewProps, ColorValue } from 'react-native';
 
-interface SeparatorPropTokens {
+export const separatorName = 'Separator';
+
+export const SeparatorInsetSpacings = [0, 16, 56, 68, 72, 108] as const;
```

Comment:
Where are these values coming from? Are they supposed to be based on some sort of design token?

Reply:
@rurikoaraki , This is directly coming from Figma design from Separator Design for Mobile. 
There will be no tokens for this as :
1. There is no ramp that supports all of this values.
2. It is values of the props that user able to pass to 'insetSpacing' - For eg : In case of the Avatar as well :  
export const AvatarSizes = [20, 24, 28, 32, 36, 40, 48, 56, 64, 72, 96, 120] as const; 
https://github.com/microsoft/fluentui-react-native/blob/3500d25b492a0fba0df3ab478b09d2775b78e452/packages/components/Avatar/src/Avatar.types.ts#L24 

There're tokens as well but for props and show typings , we have use it like this. 
One more example would be https://github.com/microsoft/fluentui-react-native/blob/3500d25b492a0fba0df3ab478b09d2775b78e452/packages/components/Checkbox/src/Checkbox.types.ts#L129